exercises/binary-search/src/test/scala/BinarySearchTest.scala

```diff
@@ -0,0 +1,74 @@
+import org.scalatest.{FunSuite, Matchers}
+
+class BinarySearchTest extends FunSuite with Matchers {
+  test("finds a value in an array with one element") {
+    pending
+    val elements = Array[Int](6)
```

Comment:
I have the [Int] there because the very last test case did not compile without the hint. I wasn't;t real jazzed about having the [Int] there. But, I didn't want to hand edit the test case to only put it where needed. I suppose I could add more logic in the generator to only put it where needed.